lib/components/app/batch-routing-panel.js

```diff
@@ -207,22 +212,18 @@ class BatchRoutingPanel extends Component {
           <UserSettings />
         */}
         <div className='desktop-narrative-container'>
```

Comment:
It might make more sense just to replace the `div` above with the new NarrativeContainer and keep the `className` prop intact.